app/celery/scheduled_tasks.py

```diff
@@ -398,6 +398,7 @@ def zendesk_new_email_branding_report():
             EmailBranding.created_at >= previous_weekday,
             User.platform_admin.is_(False),
         )
+        .order_by(EmailBranding.created_at)
```

Comment:
we could order it first by user, and then by created at maybe? Then we would cluster brandings uploaded by the same user together 🤔

Reply:
I think, feel free to make that PR if you think it's useful, but I will not for now :) Just to get this in easily.